specs/DevSpecs/telemetry.md

```diff
@@ -0,0 +1,187 @@
+# Input.Rating Telemetry Design Specification 
+ 
+## Abstract 
+This design specification covers telemetry integration into the `Input.Rating` control, which will be included in a future release of Adaptive Cards. Telemetry integration is currently slated only for the JavaScript platform; however, this developer specification will be general enough to cover all platforms supported by Adaptive Cards. This telemetry design will focus on sending telemetry via the 1DS platform, but the design is flexible enough to allow for possible integration of other telemetry platforms in the future. See more at the GitHub issue [3058](https://github.com/microsoft/AdaptiveCards/issues/3058).
+ 
+## 1. Feature Dependencies 
+### 1.1 Dependencies on other features 
+Because telemetry will be baked into the `Input.Rating` control as a result of the implementation of this design, this feature is inherently dependent on the `Input.Rating` control. Externally, this feature is dependent on the functionality of the 1DS SDK. 
+ 
+### 1.2 Other features' dependencies on this feature 
+There are currently no other existing features that directly depend on this feature. However, future telemetry integration for other elements will be based on the design presented here. 
+ 
+## 2. Platform-Specific Considerations 
+UWP, JS, and .NET are all currently supported by 1DS, allowing for relatively seamless integration of telemetry for these platforms. Mobile platforms (ie., Android and iOS) are not fully supported by 1DS, so future communication and cooperation with the 1DS team to achieve support for these platforms is necessary. 
+ 
+## 3. Performance Implications 
```

Comment:
There may be some memory/binary size etc footprint analysis that we may need to do once we start to integrate with partners like Outlook/Cortana and any expectations around framework binary sizes on different platforms etc. But that will be tested and analyzed at a later stage as part of integration and perf testing.